### ISSUE TITLE
attention score

### ISSUE THREAD

**Issue by Ad-Hua** [Atlas200dk/sample-imageinpainting-HiFill#12]
Hi! Thanks for your great work! But when I implementing the post process, I have no idea what the attention score is in your demo. In the paper, it said only one patch outside mask and one inside will have similarity. So I make all others in zero and do softmax to get the attention score. But it seems not the one in your demo, do you have any advices for me?

**Comment by duxingren14** [comment]
Yes, after softmax the weights of the inner patches are close to zero. Please find more details here.

https://github.com/duxingren14/Hifill-tensorflow
